sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -397,6 +398,18 @@ public void captureSession(final @NotNull Session session, final @Nullable Objec
     return sentryId;
   }
 
+  @Override
+  public void captureSessions(final @NotNull Sessions sessions) {
+    try {
+      transport.send(
+          new SentryEnvelope(
+              new SentryEnvelopeHeader(new SentryId(), options.getSdkVersion()),
+              SentryEnvelopeItem.fromSessionAggregates(options.getSerializer(), sessions)));
+    } catch (IOException e) {
+      options.getLogger().log(SentryLevel.WARNING, e, "Capturing session aggregates failed.");
```

Comment:
Sounds like it's an error in this case and we should investigate:

```suggestion
      options.getLogger().log(SentryLevel.ERROR, e, "Capturing session aggregates failed.");
```